### PR TITLE
Bump Go to 1.9.3

### PR DESCRIPTION
**Pull request by thaJeztah**
release notes: https://golang.org/doc/devel/release.html#go1.9.minor

full diff: https://github.com/golang/go/compare/go1.9.2...go1.9.3

### CODE REVIEW

File: Dockerfile.e2e [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## Step 1: Build tests
-FROM golang:1.9.2-alpine3.6 as builder
+FROM golang:1.9.3-alpine3.6 as builder
```
Comment (thaJeztah) [inline Dockerfile.e2e]: This one isn't available yet 😅

**Comment by AkihiroSuda** [comment]
LGTM when https://github.com/docker-library/golang/pull/201 gets merged

File: Dockerfile.e2e (hunk 1 above)
Comment (cpuguy83) [reply Dockerfile.e2e]: It's merged but not up on hub yet.

**Review by boaz0 — APPROVED** [review]
LGTM 👍

File: Dockerfile.e2e (hunk 1 above)
Comment (cpuguy83) [reply Dockerfile.e2e]: And it's up now.

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**